### PR TITLE
docs/COREDUMP_PACKAGE_METADATA.md: Add debuginfod key

### PR DESCRIPTION
**Pull request by bluca**
Signed-off-by: Mark Wielaard <mark@klomp.org>

Source: https://sourceware.org/pipermail/elfutils-devel/2021q2/003784.html (with some small rewording)

### CODE REVIEW

File: docs/COREDUMP_PACKAGE_METADATA.md [hunk 1]
```diff
@@ -58,6 +61,7 @@ Value: a JSON string with the structure described below
      "name":"coreutils",
      "version": "4711.0815.fc13.arm32",
      "osCpe":               # A CPE name for the operating system, `CPE_NAME` from os-release is a good default
+     "debuginfod": "https://debuginfod.fedoraproject.org"
```
Comment (poettering) [inline docs/COREDUMP_PACKAGE_METADATA.md]: no `/` suffix in the URL?

shouldn#t the field be `debuginfoURL` or even `debugInfoURL` or so? (or `debugInfoUrl`? i.e. "debuginfod" is a project name to me, but we don't want to refer to that, but to the link where to find the stuff

Comment (bluca) [reply docs/COREDUMP_PACKAGE_METADATA.md]: fixed as requested

**Comment by poettering** [comment]
hmm, the spec currently only provides an example with the keys, but i think we should alos have a brief explanation in prose of each key. i mean, yes, many are self-descriptive, but I still think we should say how they should be generated and from what initialized

**Comment by bluca** [comment]
> hmm, the spec currently only provides an example with the keys, but i think we should alos have a brief explanation in prose of each key. i mean, yes, many are self-descriptive, but I still think we should say how they should be generated and from what initialized

Added a table with key, example value and description

File: docs/COREDUMP_PACKAGE_METADATA.md [hunk 2]
```diff
@@ -58,6 +61,7 @@ Value: a JSON string with the structure described below
      "name":"coreutils",
      "version": "4711.0815.fc13.arm32",
      "osCpe":               # A CPE name for the operating system, `CPE_NAME` from os-release is a good default
+     "debugInfoUrl": "https://debuginfod.fedoraproject.org/",
```
Comment (williamdes) [inline docs/COREDUMP_PACKAGE_METADATA.md]:
```suggestion
     "debugInfoUrl": "https://debuginfod.fedoraproject.org/"
```
In JSON the trailing `,` does not work

**Review by williamdes — APPROVED** [review]

**Pull request merged**